tools/jit/gen_jit_dispatch.py

```diff
@@ -100,6 +100,8 @@ def is_jit_op(decl):
     # Only support a single TensorList arg
     if sum(arg['simple_type'] == 'TensorList' for arg in arguments) > 1:
         return False
+    if any(arg['simple_type'] == 'std::string' for arg in arguments):
```

Comment:
these changes can be removed as well. cc: @jamesr66a

Reply:
This change is not yet in master. Should this be removed?

Reply:
Remove